source/docs/contributing/wpilib/index.rst

```diff
@@ -7,6 +7,16 @@ Developing with allwpilib
 
 This is a list of links to the various documentation for the `allwpilib <https://github.com/wpilibsuite/allwpilib>`__ repository.
 
+Quick Start
+-----------
+
+Below is a list of instructions that guide you through cloning, building, publishing and using local allwpilib binaries in a robot project. This quick start is not intended as a replacement for the information that is further listed in this document.
+
+* Clone the repository with `git clone https://github.com/wpilibsuite/allwpilib.git`
+* Build the repository with `./gradlew build` or `./gradlew build --build-cache` if you have an internet connection
+* Publish the artifacts locally by running `./gradlew publish`
+* `Update your <https://github.com/wpilibsuite/allwpilib/blob/main/OtherVersions.md>`__ `build.gradle` to use the artifacts
```

Comment:
The link isn't very clear. Maybe make it the whole sentence or add the link after.

Reply:
Yep. I did this in the main allwpilib contribution but forgot here.